### DISCUSSION TITLE
最近两天不论玩什么整合包第一次都能玩，关掉游戏之后第二天再玩就启动不了了

### DISCUSSION THREAD

**Discussion by a8791881** [Meloong-Git/PCL#4068] [提问] [answered]
最近两天不论玩什么整合包第一次都能玩，关掉游戏之后第二天再玩就启动不了了，有时候是进地图的时候无法进入，有时候甚至PCL启动游戏都无法启动，显示的内容是：由于night config存在问题，导致了游戏崩溃，你可以尝试安装night config fixes模组，这或许能解决问题，但是尝试过了之后，确实无法解决问题，不同的包都试过了，都无法正常游玩，以下采取其中一个整合包的错误报告，希望能予以解决，谢谢。
[错误报告-2024-6-20_10.35.18.zip](https://github.com/user-attachments/files/15907657/-2024-6-20_10.35.18.zip)

**Comment by SqrtTan** [comment]
请去开issues(=_=

**Comment by MoYuan-CN** [comment]
尝试移除 Night Config。这**不是 PCL 导致**的问题

**Comment by hejiehao** [comment]
安装 https://www.mcmod.cn/class/9007.html

**Reply by SqrtTan** [reply]
原文已安装该模组无法解决
> 由于night config存在问题，导致了游戏崩溃，你可以**尝试安装night config fixes模组**，这或许能解决问题，但是**尝试过了之后**，确实无法解决问题

**Comment by LTCatt** [comment, marked as answer]
错误分析有问题，reopen as issue
